monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -73,11 +108,26 @@ if [ "$PROM_OPERATOR_CRD_UPDATE" == "true" ]; then
   log_verbose "Updating Prometheus Operator custom resource definitions"
   crds=( alertmanagerconfigs alertmanagers prometheuses prometheusrules podmonitors servicemonitors thanosrulers probes )
   for crd in "${crds[@]}"; do
-    crdURL="https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/$PROM_OPERATOR_CRD_VERSION/example/prometheus-operator-crd/monitoring.coreos.com_$crd.yaml"
+    
+    ## Determine CRD URL - if in an airgap environment, look for them in USER_DIR.
+    if [ "$AIRGAP_DEPLOYMENT" == "true" ]; then
+      crdURL=$USER_DIR/monitoring/prometheus-operator-crd/$PROM_OPERATOR_CRD_VERSION/monitoring.coreos.com_$crd.yaml
+
+      ## Fail if the CRDs could not be located.
+      if [ ! -f "$crdURL" ]; then
+        log_error "Unable to locate file: monitoring.coreos.com_$crd.yaml in"
+        log_error "$USER_DIR/monitoring/prometheus-operator-crd/$PROM_OPERATOR_CRD_VERSION directory"
+        log_error "Please make sure to provide all Prometheus Operator CRDs before running the deployment"
+        exit 1
```

Comment:
I wonder if we should move this validation that the CRD files are in the USER_DIR to the common.sh or airgap-include.sh.  Basically, check early so we can fail early and not fail after partially deploying stuff.  Seems like a better user experience.